OpenKeychain/build.gradle

```diff
@@ -43,8 +43,9 @@ dependencies {
     compile "com.splitwise:tokenautocomplete:1.3.3@aar"
     compile 'se.emilsjolander:stickylistheaders:2.6.0'
     compile 'org.sufficientlysecure:html-textview:1.1'
-    compile 'com.mikepenz.materialdrawer:library:2.8.2@aar'
-    compile 'com.mikepenz.iconics:library:0.9.1@aar'
+    compile('com.mikepenz:materialdrawer:3.0.7@aar') {
+        transitive = true
```

Comment:
transitive dependencies here makes it difficult to verify sha256 checksums later. materialdrawer depends on appcompat-v7:22.2.0 and com.mikepenz:iconics:1.0.2 (see https://repo1.maven.org/maven2/com/mikepenz/materialdrawer/3.0.7/materialdrawer-3.0.7.pom )
Please include iconics and update appcompat-v7